iac/terraform/aws/api_gw.tf

```diff
@@ -79,13 +79,14 @@ resource "aws_lambda_permission" "lambda_invoke_permission" {
   source_arn = "${aws_api_gateway_rest_api.rest_api.execution_arn}/*/*"
 }
 
-#resource "aws_api_gateway_domain_name" "api_gw_domain_name" {
-#  domain_name     = "${var.environment}.${var.bloodconnect_domain}"
-#}
-#
-#resource "aws_api_gateway_base_path_mapping" "api_mapping" {
-#  api_id      = aws_api_gateway_rest_api.rest_api.id
-#  stage_name  = aws_api_gateway_deployment.api_deployment.stage_name
-#  domain_name = aws_api_gateway_domain_name.api_gw_domain_name.domain_name
-#  base_path   = "api"
-#}
+resource "aws_api_gateway_domain_name" "api_domain" {
+  domain_name = local.apigateway-domain
+  certificate_arn = data.aws_acm_certificate.certificate.arn
+  security_policy = "TLS_1_2"
```

Comment:
should also include tls 1.3

Reply:
I spent some time after watching your comment to know the differences between `TLS 1.2` and `TLS 1.3`. It would be better if we could use `TLS 1.3` here but unfortunately `aws_api_gateway_domain_name` currently doesn't support `TLS 1.3` as their `security_policy`.
Reference: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_domain_name#security_policy

Reply:
Strange that we're using api gateway domain since we already have cloudfront

Reply:
The decision is upgraded. Will implement on code soon.

Reply:
What do you mean